### DISCUSSION TITLE
Disable Cross-Hatching?

### DISCUSSION THREAD

**Discussion by UD551** [supermerill/SuperSlicer#2253] [Q&A] [answered]
Hello Again Everyone,
Thank you for taking time of your day to respond to my issues. 

I am currently trying to disable cross-hatching on the first layer. However after trying a couple of print setting modifications, I was not able to achieve my goal. 
The only thing I can think of is using height range modifiers, but It is not working when I select each individual layer. My first layer is only made up of solid bottom layers using solid infill. I am currently not using any top solid layers.  
I would greatly appreciate any guidance and help from the Super Slicer team. 
Thank you in advance. 

PS: I am using the newest version of super slicer.

**Comment by supermerill** [comment, marked as answer]
> The only thing I can think of is using height range modifiers, but It is not working when I select each individual layer. My first layer is only made up of solid bottom layers using solid infill. I am currently not using any top solid layers.

I think you can avoid this problem by unticking the `Ensure vertical shell thickness`

what do you mean by "cross-hatching"? making the first and second layer infill orientation the same?
If you want that for only one layer, you have to use a height range modifier. If you want that for all layer, you can use the infill->angle->increment field with 90° value

**Reply by UD551** [reply]
Thank you!
